<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QWidget" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="profileIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../assets/image/mario.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>166</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt</string>
   </property>
   <property name="text">
    <string>Michel Pignouf</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="_nameFriend" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>200</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameFriend">
      <property name="styleSheet">
       <string notr="true">font: 18pt</string>
      </property>
      <property name="text">
       <string>Veigo la Veigs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>351</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Friends">
    <attribute name="title">
     <string>Friends</string>
    </attribute>
    <widget class="QListWidget" name="userList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Groups">
    <attribute name="title">
     <string>Groups</string>
    </attribute>
    <widget class="QListWidget" name="groupList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="editProfile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>165</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Profile</string>
   </property>
  </widget>
  <widget class="QWidget" name="_imageFriend" native="true">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="imageFriend">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../assets/image/mario.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="disconnect">
   <property name="geometry">
    <rect>
     <x>1418</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="friendBar">
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="placeholderText">
       <string>Search </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addFriend">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Add friend</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_call" native="true">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>340</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QListWidget" name="callList">
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_statut" native="true">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="statut">
      <property name="text">
       <string>Statut:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statutIcon">
      <property name="maximumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../assets/image/co.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_groupList" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QListWidget" name="grpList">
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_invitGroup" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="_invitGroupL">
    <item>
     <widget class="QLineEdit" name="inviteBar">
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="placeholderText">
       <string>Search </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addToGroup">
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Invite</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="groupBar">
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="placeholderText">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createGroup">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Create group</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_enterCall" native="true">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>260</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="enterCall">
      <property name="styleSheet">
       <string notr="true">border: 0px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_quitCall" native="true">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>260</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QPushButton" name="quitCall">
      <property name="styleSheet">
       <string notr="true">border: 0px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../test.qrc"/>
  <include location="../../../../../../test.qrc"/>
 </resources>
 <connections/>
</ui>
